priv/repo/migrations/20251204170944_nullable_jwt_secrets.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Realtime.Repo.Migrations.NullableJwtSecrets do
+  use Ecto.Migration
+
+  def change do
+    alter table(:tenants) do
+      modify :jwt_secret, :string, null: true
+    end
+
+    create constraint(:tenants, :jwt_secret_or_jwt_jwks_required,
```

Comment:
Do we want to add this to the [changeset](https://hexdocs.pm/ecto/Ecto.Changeset.html#check_constraint/3)?